custom_components/foxess_modbus/entities/xx1_aux_entity_descriptions.py

```diff
@@ -46,6 +50,18 @@
         scale=0.001,
         validate=[Range(0, 10000)],
     ),
+    ModbusIntegrationSensorDescription(
+        key="pv1_power_total",
+        address=11002,
```

Comment:
Do we have "power" in the names of other sensors which denote energy? Maybe "pv1_sum_total" to match the solar_sum_total? Or is there a sensor from StealthChestnut's integration we want to copy the name of?

Reply:
I matched against previous convention - `key="feed_in_sum_total"` StealthChestnuts is `pv1_sum` which isn't descriptive enough IMHO - sum of power? voltage? current?

Reply:
Hmm. I guess `pv1_energy_total` is strictly correct, but goes against what we have for feed_in etc which don't specify whether it's energy/power (and use "sum" to imply its energy). I guess "sum" on its own could mean charge, which is the integral of current...

Reply:
They were only released under beta so I think there's an opportunity to rename. Suggest:

`feed_in_energy_today`
`feed_in_energy_total`
...
`pv1_energy_total`

Reply:
Yeah I like the sound of those. Thanks!